media/src/commonMain/kotlin/PlaybackStateRepo.kt

```diff
@@ -64,6 +64,16 @@ open class PlaybackStateRepo(
                 when (event) {
                     is MediaManager.Event.DidPrepare -> takeAndChangeState(PlaybackState.Initialized::class) { it.prepared(event.playableMedia) }
                     is MediaManager.Event.DidFailWithError -> takeAndChangeState(PlaybackState.Active::class) { it.failWithError(event.error) }
+                    is MediaManager.Event.RateDidChange -> takeAndChangeState(PlaybackState.Started::class) { state ->
+                        when (event.newRate) {
+                            0.0f -> when (state) {
+                                is PlaybackState.Playing -> state.pause
```

Comment:
I see playing goes to pause when rate becomes zero, but then rate goes from zero to non zero value we have to handle pause -> play as well?